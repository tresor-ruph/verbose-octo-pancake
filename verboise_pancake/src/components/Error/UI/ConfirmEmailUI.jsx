import React from "react";
import MainHeader from "components/Navbars/MainHeader";

const ConfirmEmailUI = (props) => {
  const { fromLoging, email, handleSendLink } = props;
  return (
    <div>
      <MainHeader />
      <div className="container bootstrap snippet">
        <div className="main-div">
          <div className="center-div">
            <p className="text-res">Confirm Account</p>
            <hr />

            <p>
              {!fromLoging
                ? `An activation EMail has been sent to ${email}. Please follow the
              mail instructions in orther to activate your account`
                : `Your account has not been activated. Please verify your mails for the activation link`}
            </p>
            <p style={{ marginTop: "50px", fontSize: "12px" }}>
              Did not recieve the Mail ?{" "}
              <a
                // href=""
                style={{ color: "#42D0ED", textDecoration: "none" }}
                onClick={() => handleSendLink()}
              >
                resend Link
              </a>
            </p>
          </div>
          <div></div>
        </div>
      </div>
    </div>
  );
};

export default ConfirmEmailUI;
