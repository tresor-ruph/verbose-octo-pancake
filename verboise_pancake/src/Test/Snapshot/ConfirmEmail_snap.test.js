import React from 'react'
import renderer from 'react-test-renderer'
import ConfirmEmailUI  from './../../components/Error/UI/ConfirmEmailUI.jsx'

it('renders correctly', () => {
    const ConfirmSnap = renderer.create(<ConfirmEmailUI />).toJSON();
    expect(ConfirmSnap).toMatchSnapshot();

});