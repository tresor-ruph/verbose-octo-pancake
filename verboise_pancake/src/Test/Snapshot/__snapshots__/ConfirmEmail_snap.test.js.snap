// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="container bootstrap snippet"
  >
    <div
      className="main-div"
    >
      <div
        className="center-div"
      >
        <p
          className="text-res"
        >
          Confirm Account
        </p>
        <hr />
        <p>
          An activation EMail has been sent to undefined. Please follow the
              mail instructions in orther to activate your account
        </p>
        <p
          style={
            Object {
              "fontSize": "12px",
              "marginTop": "50px",
            }
          }
        >
          Did not recieve the Mail ?
           
          <a
            onClick={[Function]}
            style={
              Object {
                "color": "#42D0ED",
                "textDecoration": "none",
              }
            }
          >
            resend Link
          </a>
        </p>
      </div>
      <div />
    </div>
  </div>
</div>
`;
